src/wwwroot/controllers/HeaderCtrl.js

```diff
@@ -26,6 +26,40 @@
     };
     $scope.getSubmenues = $rootScope.getSubmenues;
     $scope.isSubmenuVisible = $rootScope.isSubmenuVisible;
+
+    UpdateTrialHeader();
```

Comment:
Is it executed on each navigation?

I think that it is fine by the moment, but probably we will change it soon to be updated periodically.